src/forms-opportunities/constants.js

```diff
@@ -31,6 +31,9 @@ export const OPPORTUNITY_LIMIT = 2;
 export const DAILY_PAGEVIEW_THRESHOLD_DEFAULT = 200;
 export const DAILY_PAGEVIEW_THRESHOLD_ALL = 7;
 
+// Form types to ignore when is_lead_gen is false
+export const FORM_TYPES_TO_IGNORE = ['search form', 'login form', 'zip code search form', 'filter form', 'otp form', 'calculator form', 'product listing/selection form', 'cart form', 'file upload form', 'payment form'];
```

Comment:
I am doubtful of this list since we are doing an exact match. Will re-visit once we see any problems